bot/exts/evergreen/stackoverflow.py

```diff
@@ -0,0 +1,45 @@
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, command, cooldown
+
+BASE_URL = "https://api.stackexchange.com/2.2/search/advanced?order=desc&sort=activity&site=stackoverflow&q={query}"
+SEARCH_URL = "https://stackoverflow.com/search?q={query}"
+SO_COLOR = 0xF98036
+
+
+class Stackoverflow(Cog):
+    """A cog which returns the top 5 results of a query from stackoverflow."""
```

Comment:
```suggestion
    """Sends the top 5 results of a query from stackoverflow."""
```
It “sends” and doesn’t “return”.
Also everyone knows it is a cog already.